### DISCUSSION TITLE
Atlas v0.3.1 Benchmarks (semi-historic)

### DISCUSSION THREAD

**Discussion by jacksyrm** [Atlas-OS/Atlas#1161] [Resources]
> [!NOTE]
> This post was originally made on November 6, 2023, but migrated from our deprecated forum. This is now semi-historic because there have been many updates to Atlas since.

We've conducted some benchmarks on VALORANT using Windows 11.

## Specifications

**CPU:** Ryzen 7 5800X3D (SMT disabled)
**CPU Cooler:** NZXT Kraken X53 AIO
**RAM:** 16GB 3600Mhz Corsair Vengeance LPX
**GPU:** MSI 2070 SUPER 8GB
**Motherboard:** MSI B550 Gaming Carbon Wi-Fi
**Storage:** WD_BLACK SN750 NVMe

## Testing Methodologies

Benchmarking tool: CapframeX
Benchmarks were taken three times with data aggregated

## Setting up each OS

**Atlas Configuration**

Windows Defender disabled
Manual Driver Installation (NVCleanstall minimal driver 546.01-desktop-win10-win11-64bit-international-dch-whql.exe)
Core Isolation disabled
CPU mitigations enabled

**Stock Configuration**

Windows Defender real-time protection disabled
Stock driver installations (NVCleanstall minimal driver 546.01-desktop-win10-win11-64bit-international-dch-whql.exe)
The power plan changed to High Performance

**Mirrored NVIDIA Display Settings**

![image](https://github.com/user-attachments/assets/b21a148d-5e37-468c-b561-90717cd1a186)

![image](https://github.com/user-attachments/assets/1582f285-dc9e-4b7f-b846-44a2518d6f14)

![image](https://github.com/user-attachments/assets/0080b356-2899-481e-b06b-b51110dc6a37)

![image](https://github.com/user-attachments/assets/154dd34c-34dc-48e9-b4be-b0fc1ff3dd80)

## Setting up game settings

All video settings low, multi-threading enabled
Exclusive Fullscreen
Nvidia Reflex - On
Beta Raw Input - On

![image](https://github.com/user-attachments/assets/0fb5de62-13b2-4ffc-82de-467723d38dbb)

## How did we test?

To keep it as identical as possible, the benchmark was taken on the map "Haven" with the same path and the same character, Brimstone. Here is a diagram of the path we took to reproduce a consistent benchmark.

![image](https://github.com/user-attachments/assets/e5cab04c-868f-4f75-ab9b-2566203fa6c7)

## The benchmark results

1% Percentile FPS: 665.3 → 800.7 (most important metric)

![image](https://github.com/user-attachments/assets/0893e24f-56cd-4100-8162-4e6d69eb676b)

![image](https://github.com/user-attachments/assets/4b19dde3-a1da-4447-9c6d-48c9487445a0)

**Comment by the-P1neapple** [comment]
> [!NOTE]
> This benchmark was originally posted on December 16, 2023, but migrated from our deprecated forum. This is now semi-historic because there have been many updates to Atlas since.

# Specifications
CPU: Ryzen 5 5600G
CPU Cooler: AMD Wraith Cooler
RAM: 16GB 3600Mhz Corsair Vengeance LPX
GPU: MSI 1650 SUPER 4GB
Motherboard: MSI B550M Pro-VDH WiFi
Storage: Crucial P1 500GB

# Testing Methodologies
I used the same testing method as Jack. Same configuration as above.
Tested using Windows 11 23H2

# Important Notes:
As my CPU is much more recent that my GPU. There is a bottleneck that can be shown in the result that does not show optimal performance improvements compared to Jack's benchmark above.

# Benchmark Results:
![1702726079-114158-export.png](https://github.com/user-attachments/assets/1ca26c56-a607-4ac8-97c0-e6bc8c0644bd)

1% Percentile FPS: 400.43 → 460.96

**Comment by authorless** [comment]
Hello, will new measurements of AtlasOS 0.4.0 tests be made in the near future?

**Reply by johngagefaulkner** [reply]
For whoever and wherever this may be applicable, I plan on (within the next few days) setting up dual-boot with whatever the latest version is as of May 18, 2025. I can provide before and after numbers for both Fortnite and Marvel Rivals.

**Reply by Ast3risk-ops** [reply]
> For whoever and wherever this may be applicable, I plan on (within the next few days) setting up dual-boot with whatever the latest version is as of May 18, 2025. I can provide before and after numbers for both Fortnite and Marvel Rivals. 

Both of those games suck ass and don't look realistic, do a CS2 benchmark (no usermode anticheat to decrease performance either, VAC is serverside).

**Reply by Assationater** [reply]
> For whoever and wherever this may be applicable, I plan on (within the next few days) setting up dual-boot with whatever the latest version is as of May 18, 2025. I can provide before and after numbers for both Fortnite and Marvel Rivals.

Yes please, where can i find the post?

**Comment by Jonah-Schepers** [comment]
Can there be a performance metric for Escape from tarkov.

**Reply by T1mJka** [reply]
nop

**Comment by marolua** [comment]
I have a question, this don't impact my pc for developpment or other we are okay ?

**Reply by Stensel8** [reply]
AtlasOS v0.5.0 can cause issues when you need to use WSL2/Docker and Virtualization, however this can be fixed by not disabling core isolation / VBS.

From my knowledge, that would be the only breaking change. There are no other issues and you're completely fine on AtlasOS :)

**Comment by nandeesh13** [comment]
If someone performs a fresh installation of AtlasOS on their PC, is Windows Defender disabled by default? If so, why is Windows Defender turned off? Shouldn't it be enabled for security reasons?

**Reply by Stensel8** [reply]
> If someone performs a fresh installation of AtlasOS on their PC, is Windows Defender disabled by default? If so, why is Windows Defender turned off? Shouldn't it be enabled for security reasons?

Users can choose if they want to keep Defender running or if they want it stripped out for additional performance.

To answer your second question: Defender indeed needs to be temporarily stopped. This is because the AME wizard will modify your OS, and Defender does not allow that. Some tweaks and optimizations go very deep, and Defender sees this as malicious.

After the playbook is applied, Defender runs fine, and there are no issues. Alternatively, it is disabled, depending on what you chose when you ran the AME wizard.

**Reply by nandeesh13** [reply]
Understood. Thanks for the info.

**Reply by Stensel8** [reply]
> Understood. Thanks for the info.

You're welcome
